docs-source/content/quickstart/install.md

```diff
@@ -24,9 +24,33 @@ subjects:
 EOF
 ```
 
+#### Use `Helm` to install the Operator and [Traefik](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/charts/traefik/README.md) load balancer. Use the [values.yaml](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/charts/traefik/values.yaml) in the sample but set `kubernetes.namespaces` specifically.
+
+Setup Helm
+
+For Helm version 2.x:
+
+```bash
+$ helm init
+
+```
+
+For Helm version 3.x:
+
+```bash
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+```
+
 #### Create a Traefik (Ingress-based) load balancer.
 
-Use `helm` to install the [Traefik](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/charts/traefik/README.md) load balancer. Use the [values.yaml](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/charts/traefik/values.yaml) in the sample but set `kubernetes.namespaces` specifically.
+Create a namespace for the load balancer
```

Comment:
We didn't have this step (create a ns for the LB) in the original instructions because the namespace creation was included in the command:
$ helm install stable/traefik \
  --name traefik-operator \
  --namespace traefik \
  --values kubernetes/samples/charts/traefik/values.yaml  \
  --set "kubernetes.namespaces={traefik}" \
  --wait 
It looks like the commands you use for creating the Traefik LB are similar to the original one (they both include --namespace traefik). Do we still need this step?

Reply:
Helm 2 automatically create the namespace if it doesn't exist but Helm 3 no longer does that,  so I added a step for both.

Reply:
Got it. Thanks.